### ISSUE TITLE
rawtherapee@5.7  Access to the path 'C:\innounp.log' is denied

### ISSUE THREAD

**Issue by richardmalone** [lukesampson/scoop-extras#3046]
Thank you for adding rawtherapee to the bucket.  

The current version tries to write an extraction log to the c:\ root directory, but on my system that requires admin rights which I do not have. 

I assume it should be pointed to %APPDATA%/tmp or something like that.

Full ps output is:

```
Out-File : Access to the path 'C:\innounp.log' is denied.
At C:\Users\rmalone\scoop\apps\scoop\current\lib\core.ps1:432 char:9
+         Out-File -FilePath $LogPath -Encoding ASCII -Append -InputObj ...
+         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : OpenError: (:) [Out-File], UnauthorizedAccessException
    + FullyQualifiedErrorId : FileOpenFailure,Microsoft.PowerShell.Commands.OutFileCommand

ERROR Exit code was 1!
Failed to extract files from RawTherapee_5.7_WinVista_64.exe.
Log file:
  \innounp.log
```

Thanks again, Richard

**Comment by Ash258** [comment]
Installation is working fine on my end.

This log should reside inside application directory, while installation.

But you are correct if I break manifest locally and then spawn installation from `C:\Windows` I get log in root

![image](https://user-images.githubusercontent.com/13260377/67379368-15686a80-f589-11e9-83b8-15657039729c.png)

/cc @niheaven

**Comment by richardmalone** [comment]
Thanks for the quick response. Tried some different things but still couldn't get it to work.

How would I unbreak my manifest locally? :)

Or is there a string I can pass that will set $LogPath for a single installation?

**Comment by Ash258** [comment]
Try to edit `$env:SCOOP\buckets\extras\bucket\rawtherapee.json` and replace line 13 with this:

`        "script": "(Get-ChildItem \"$dir\" 'RawTherapee*WinVista*.exe').Fullname | Expand-InnoArchive -DestinationPath \"$dir\""
`

and then install again.

Or use powershell core for installation. I tested inside pwsh not powershell 5, which changed default to string of gci items and could cause this.

**Comment by richardmalone** [comment]
Great, that did it. Thank you.

Will ask an administrator to install powershell core.

Richard

**Comment by Ash258** [comment]
No need to install pwsh if not have. This issue will be fixed for powershell 5 in #3047 

This is just my bad when I missed this potential use case.

Do not forget to change manifest inside bucket folder back otherwise you will see errors on scoop update.

`cd $env:SCOOP\buckets\extras; git reset --hard`
